packages/backend/src/apps/formatter/actions/date-time/transforms/convert-date-time/fields.ts

```diff
@@ -0,0 +1,71 @@
+import type { IFieldDropdown } from '@plumber/types'
+
+import z from 'zod'
+
+import { ensureZodEnumValue, ensureZodObjectKey } from '@/helpers/zod-utils'
+
+const formatStringsEnum = z.enum([
+  'dd/LL/yy',
+  'dd/LL/yyyy',
+  'dd LLL yyyy',
+  'dd LLLL yyyy',
+  'hh:mm a',
+  'hh:mm:ss a',
+  'dd LLL yyyy hh:mm a',
+  'dd LLL yyyy hh:mm:ss a',
+])
+
+export const fieldSchema = z.object({
+  formatDateTimeToFormat: formatStringsEnum,
+})
+
+export const field = {
```

Comment:
question: all these formats are data points that Stacey collected or is it a gut feel on what possible formats users may require?

Reply:
mix of gut feel and some data points. gut feel are the am/pm ones